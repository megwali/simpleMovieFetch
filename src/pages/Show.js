const Show = () => (
  <div>
    <h2>Showie</h2>
  </div>
);

export default Show;
