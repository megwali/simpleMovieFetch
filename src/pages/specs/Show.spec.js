import { create } from 'react-test-renderer';
import Show from '../Show';

describe('Show', () => {
  it('renders', () => {
    const component = create(<Show />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
