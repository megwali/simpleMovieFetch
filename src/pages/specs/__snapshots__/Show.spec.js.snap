// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Show renders 1`] = `
<div>
  <h2>
    Showie
  </h2>
</div>
`;
